### ISSUE TITLE
[Calendar] Screen reader reads each calendar date twice

### ISSUE THREAD

**Issue by yanbxice** [microsoft/fluentui#13795]
### Environment Information

- **Package version(s)**: I'm using 6.190.0 but can also repro on Fluent UI site
- **Browser and OS versions**: Edge and Narrator

### Describe the issue:

This issue happens on a day picker with month picker on the right. When I navigate on the day picker by swiping on the touch screen, each date was read twice. Arrow keys navigation works fine.

### Please provide a reproduction of the issue in a codepen:
This can be reproduced on the Fluent UI website: https://developer.microsoft.com/en-us/fluentui#/controls/web/calendar, under section "Inline Calendar with month picker and overlaid year picker when month header is clicked"

#### Actual behavior:

Each date is read twice whiling navigating

#### Expected behavior:

Date should be read only once.

**Comment by lorejoh12** [comment]
This is a dupe of #11140, which was closed as being out of scope.

**Comment by lorejoh12** [comment]
closing as dupe.
